### PR TITLE
chore: linting and unit-test

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

- linting for: #1397
- unit-test for: https://github.com/getAlby/lightning-browser-extension/commit/31018f73294c242f1a3534eb8a9c749fc07da800

### CODE REVIEW

File: src/app/screens/Onboard/TestConnection/index.tsx [hunk 1]
```diff
@@ -49,11 +49,13 @@ export default function TestConnection() {
       setLoading(false);
       clearTimeout(timer);
     }
-  }
+  }, [t]);
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.tsx:52]: why was this not needed before? 
did you test this very well? (I had strange errors in some places when I had to do this callback,useEffect stuff - and I still don't understand it) (e.g. did you test the timeout?)
you are missing the test connection for the options page.

Comment (escapedcat) [reply src/app/screens/Onboard/TestConnection/index.tsx:52]: > why was this not needed before?

Define "before"

> did you test this very well? (I had strange errors in some places when I had to do this callback,useEffect stuff - and I still don't understand it) (e.g. did you test the timeout?) 

Let me test. Just opened this to see if the pipeline will be green.

> you are missing the test connection for the options page.

What do you mean? With this PR I just wanted to fix the warning.

Comment (bumi) [reply src/app/screens/Onboard/TestConnection/index.tsx:52]: please always make sure that things work when you change thing on such critical components. They simply have to work, if it's broken for some reason then people will have issues adding accounts. Checking the pipeline is (and will never be) enough.
This also helps you to understand what goes on and how things are used. 

So here for example you seem to miss that there are two similar flows (welcome + options) both have a test-connection screen - and sadly are not very DRY. So you have to make this change in the other screen, too.

Comment (escapedcat) [reply src/app/screens/Onboard/TestConnection/index.tsx:53]: Tested the welcome/onboard testconnection with this and it seems to work well.  
The same way is not working for the options screen so I'm leaving that for now and will have a look with Lisa when she's back.
Would you mind testing this as well?

File: src/app/screens/Onboard/TestConnection/index.tsx [hunk 2]
```diff
@@ -47,13 +47,14 @@ export default function TestConnection() {
       setErrorMessage(message);
     } finally {
       setLoading(false);
+      setErrorMessage("");
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.tsx:50]: why that? finally runs always (no matter if it is successful or an error happens). So does this not overwrite the `setErrorMessage` in the error case in line 47?

Comment (escapedcat) [reply src/app/screens/Onboard/TestConnection/index.tsx]: True, I just noticed that if the timeout error is showing and a success is happening during that, error and success show at the same time. Adjusted it.

**Comment by bumi** [comment]
so imo this is broken. It does no longer show an error message and I also don't get why we make this change in one of the components (onboard) and not the other one (options).  
it seems like we don't know what we're doing here? so I'd rather not change this.

**Pull request closed without merging**